config/crd/bases/appprotect.f5.com_appolicies.yaml

```diff
@@ -161,61 +174,114 @@ spec:
-                                  - VIOL_BLACKLISTED_IP
-                                  - VIOL_COOKIE_EXPIRED
-                                  - VIOL_COOKIE_LENGTH
-                                  - VIOL_COOKIE_MALFORMED
-                                  - VIOL_COOKIE_MODIFIED
-                                  - VIOL_DATA_GUARD
-                                  - VIOL_ENCODING
-                                  - VIOL_EVASION
-                                  - VIOL_FILETYPE
-                                  - VIOL_FILE_UPLOAD
-                                  - VIOL_FILE_UPLOAD_IN_BODY
-                                  - VIOL_HEADER_LENGTH
-                                  - VIOL_HEADER_METACHAR
-                                  - VIOL_HTTP_PROTOCOL
-                                  - VIOL_HTTP_RESPONSE_STATUS
-                                  - VIOL_JSON_FORMAT
-                                  - VIOL_JSON_MALFORMED
-                                  - VIOL_JSON_SCHEMA
-                                  - VIOL_MANDATORY_PARAMETER
-                                  - VIOL_MANDATORY_REQUEST_BODY
-                                  - VIOL_METHOD
-                                  - VIOL_PARAMETER
-                                  - VIOL_PARAMETER_DATA_TYPE
-                                  - VIOL_PARAMETER_EMPTY_VALUE
-                                  - VIOL_PARAMETER_LOCATION
-                                  - VIOL_PARAMETER_MULTIPART_NULL_VALUE
-                                  - VIOL_PARAMETER_NAME_METACHAR
-                                  - VIOL_PARAMETER_NUMERIC_VALUE
-                                  - VIOL_PARAMETER_REPEATED
-                                  - VIOL_PARAMETER_STATIC_VALUE
-                                  - VIOL_PARAMETER_VALUE_LENGTH
-                                  - VIOL_PARAMETER_VALUE_METACHAR
-                                  - VIOL_POST_DATA_LENGTH
-                                  - VIOL_QUERY_STRING_LENGTH
-                                  - VIOL_RATING_THREAT
-                                  - VIOL_RATING_NEED_EXAMINATION
-                                  - VIOL_REQUEST_MAX_LENGTH
-                                  - VIOL_REQUEST_LENGTH
-                                  - VIOL_THREAT_CAMPAIGN
-                                  - VIOL_URL
-                                  - VIOL_URL_CONTENT_TYPE
-                                  - VIOL_URL_LENGTH
-                                  - VIOL_URL_METACHAR
-                                  - VIOL_XML_FORMAT
-                                  - VIOL_XML_MALFORMED
+                                  - "VIOL_ACCESS_INVALID"
+                                  - "VIOL_ACCESS_MALFORMED"
+                                  - "VIOL_ACCESS_MISSING"
+                                  - "VIOL_ASM_COOKIE_HIJACKING"
+                                  - "VIOL_ASM_COOKIE_MODIFIED"
+                                  # - "VIOL_ATTACK_SIGNATURE"
```

Comment:
@fabriziofiorucci any reason commented policy violations have been left in?

Reply:
@vepatel those are lines that came with the CRD used in the internal test repo, as provided by @schweits so I didn't change those

Reply:
Those are all valid values for that field, but the commented-out violations are for features that aren't supported by KIC yet. They can probably be removed here if they're not wanted

Reply:
@fabriziofiorucci can you please remove them and rebase? I can approve the PR as well then 👍🏼